### ISSUE TITLE
mask result with white line problem.

### ISSUE THREAD

**Issue by suchot** [ZhengPeng7/BiRefNet#146]
During fine-tuning， using the test.sh script to evaluate models.  The mask result always has some white line. Does this result meet expectations?  Do you have any intuition to solve this problem?
![image](https://github.com/user-attachments/assets/d7c8961d-8c71-4a48-9855-8a20ab576eb8)

**Comment by ZhengPeng7** [comment]
We didn't see this kind of problem in previous training.
You can have a check on your training data. If it is hard to find the reason, try training in some standard benchmarks (e.g., DIS5K) with only 50 epochs to have quick feedback.
